third_party/move/move-vm/integration-tests/src/tests/module_storage_tests.rs

```diff
@@ -304,112 +280,3 @@ fn test_verified_script_caching() {
         .assert_cached_state(vec![], vec![&a_id, &b_id, &c_id]);
     code_storage.assert_cached_state(vec![], vec![&hash]);
 }
-
-#[test]
-fn test_function_value_extension() {
```

Comment:
what's the story of this test?

Reply:
This was used for function value extension function I removed only. Basically, for function values I first added `get_function_arg_tys`, but then the decision was to store layouts, so it was not needed any longer.